src/browser/documentation/help/create-database.jsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react'
+import ManualLink from 'browser-components/ManualLink'
+import DbsOnSystemDb from './partials/dbs-on-systemdb'
+const title = 'CREATE DATABASE'
+const subtitle = 'Create a new database'
+const category = 'administration'
+const content = (
+  <>
+    <p>
+      Databases can be created using <code>CREATE DATABASE</code>
```

Comment:
Looking at the pattern used elsewhere in this directory, the structure seems to be along the lines of 
`The <code>CREATE DATABASE</code> clause/command will...`

Reply:
You're right. I actually had a hard time figuring out what to write here and ended up copying what's in the docs. I'm still not sure if there's even the need for a description in some of these commands to be honest, as it's not giving any new information, the command itself is self explanatory.

Reply:
I rephrased the descriptions, but I'm still not sure if all of them are needed as I mentioned above. Would love to hear @oskarhane's input on this.